docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 
-## Posting pull requests
+### Closing issues
+
+If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
+
+## Pull requests
+
+### Posting pull requests
 
 If you are posting a pull request, please:
 
+* Find if your pull request solves an existing issue. Note that just because an issue is open does not guarantee that it is something we will accept, as the number of issues we receive means that not all of them have been thoroughly discussed, and there may be open issues for features that we decide we do not want.
 * Do not combine multiple unrelated changes.
 * Check the diff and make sure the pull request does not contain unintended changes.
 * If changing the C++ code, follow the [coding standard](https://endless-sky.github.io/styleguide/styleguide.xml).
+* Fill out the pull request template to the best of your ability. GitHub's pull request template features are not as thorough as issue templates. You're able to delete the entire pull request template and put in whatever you want; please do not do this. The purpose of the pull request template is to help facilitate the review, approval, and merging of pull requests.
 
 If proposing a major pull request, start by posting an issue and discussing the best way to implement it. Often the first strategy that occurs to you will not be the cleanest or most effective way to implement a new feature.
 
-## Closing issues
+### What to expect when opening a pull request
 
-If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
+Endless Sky is developed by a community of volunteers. We're a collection of people with various different interests and skill sets, so the type of pull request you open will influence who will be looking at it.
+
+All pull requests go through a review process before being merged. We have a group of dedicated Reviewers who help provide reviews to pull requests as they come in. We also have a group of Developers who make determinations on what does and does not get merged, and are the ones who have the access to merge pull requests.
+* For smaller pull requests, such as typo fixes, it may only require a Developer seeing your pull request in order for it to be merged.
+* Larger pull requests typically receive reviews from multiple Reviewers and Developers before they are merged. That, in addition to the current backlog of open pull requests, can result in it taking months before a larger pull request is merged. We ask for your patience when waiting for your pull request to be reviewed.
```

Comment:
Language to better communicate that you're not just waiting for *a* one-and-done review and then merge. That there's a *process* (i.e. potentially many steps) to be performed.
```suggestion
* Larger pull requests typically receive reviews from multiple Reviewers and Developers before they are merged. That, in addition to the current backlog of open pull requests, can result in the full review process taking a number of months before a larger pull request may be merged. We ask for your patience while your pull request undergoes the review process.
```